ext/standard/basic_functions.c

```diff
@@ -2624,3 +2624,26 @@ PHP_FUNCTION(sys_getloadavg)
 }
 /* }}} */
 #endif
+
+PHP_FUNCTION(cpu_cores)
```

Comment:
Nit: We might want to incorporate `num`, `cpu_cores` sounds like it might provide more information, like `cpu_num_cores`. But admittedly it doesn't sound as nice.

Reply:
True. I found https://docs.rs/num_cpus/latest/num_cpus/ and thought it was a good name.